src/loggly.tracker.min.map

```diff
@@ -1 +1 @@
\ No newline at end of file
+{"version":3,"file":"loggly.tracker.min.js","sources":["loggly.tracker.js"],"names":["window","document","uuid","replace","c","r","Math","random","v","toString","LogglyTracker","this","key","sendConsoleErrors","setKey","tracker","setSession","setInputUrl","setSendConsoleError","_onerror","onerror","msg","url","line","col","push","category","exception","message","lineno","colno","apply","arguments","inputUrl","LOGGLY_INPUT_PREFIX","logglyCollectorDomain","LOGGLY_COLLECTOR_DOMAIN","LOGGLY_INPUT_SUFFIX","location","protocol","LOGGLY_SESSION_KEY","LOGGLY_SESSION_KEY_LENGTH","length","prototype","session_id","setCookie","readCookie","data","type","self","text","undefined","logglyKey","track","sessionId","im","Image","q","encodeURIComponent","JSON","stringify","src","ex","console","log","cookie","i","indexOf","end","slice","value","existing","_LTracker","eLength"],"mappings":"CAAA,SAAUA,EAAQC,GAOd,QAASC,KAEL,MAAO,uCAAuCC,QAAQ,QAAS,SAASC,GACpE,GAAIC,GAAkB,GAAdC,KAAKC,SAAY,EAAGC,EAAS,KAALJ,EAAWC,EAAO,EAAFA,EAAM,CACtD,OAAOG,GAAEC,SAAS,MAI1B,QAASC,KACLC,KAAKC,KAAM,EACXD,KAAKE,mBAAoB,EAG7B,QAASC,GAAOC,EAASH,GACrBG,EAAQH,IAAMA,EACdG,EAAQC,aACRC,EAAYF,GAGhB,QAASG,GAAoBH,EAASF,GAGlC,GAFAE,EAAQF,kBAAoBA,EAEzBE,EAAQF,qBAAsB,EAAM,CACnC,GAAIM,GAAWnB,EAAOoB,OAEtBpB,GAAOoB,QAAU,SAAUC,EAAKC,EAAKC,EAAMC,GACvCT,EAAQU,MACJC,SAAU,qBACVC,WACIC,QAASP,EACTC,IAAKA,EACLO,OAAQN,EACRO,MAAON,KAIXL,GAAgC,kBAAbA,IACnBA,EAASY,MAAM/B,EAAQgC,aAMvC,QAASf,GAAYF,GACjBA,EAAQkB,SAAWC,GACZnB,EAAQoB,uBAAyBC,GAClC,WACArB,EAAQH,IACRyB,EAtDV,GAAIH,GAAsB,QAAY,WAAajC,EAASqC,SAASC,SAAW,IAAM,IAAQ,MAC1FH,EAA0B,qBAC1BC,EAAsB,QACtBG,EAAqB,wBACrBC,EAA4BD,EAAmBE,OAAS,CAqD5DhC,GAAciC,WACV3B,WAAY,SAAS4B,GACdA,GACCjC,KAAKiC,WAAaA,EAClBjC,KAAKkC,UAAUlC,KAAKiC,aACbjC,KAAKiC,aACZjC,KAAKiC,WAAajC,KAAKmC,aACnBnC,KAAKiC,aACLjC,KAAKiC,WAAa1C,IAClBS,KAAKkC,UAAUlC,KAAKiC,eAIhCnB,KAAM,SAASsB,GACX,GAAIC,SAAcD,EAElB,IAAKA,IAAmB,WAATC,GAA8B,WAATA,GAApC,CAIA,GAAIC,GAAOtC,IAGf,IAAY,WAATqC,EACKD,GACIG,KAAQH,OAET,CACH,GAAGA,EAAKZ,sBAEJ,YADAc,EAAKd,sBAAwBY,EAAKZ,sBAQtC,IAJ8BgB,SAA3BJ,EAAKlC,mBACJK,EAAoB+B,EAAMF,EAAKlC,mBAGhCkC,EAAKK,UAEJ,WADAtC,GAAOmC,EAAMF,EAAKK,UAItB,IAAGL,EAAKH,WAEJ,WADAK,GAAKjC,WAAW+B,EAAKH,YAKzBK,EAAKrC,KAITqC,EAAKI,MAAMN,KAGfM,MAAO,SAASN,GAEZA,EAAKO,UAAY3C,KAAKiC,UAEtB,KACI,GAAIW,GAAK,GAAIC,OACbC,EAAI,aAAeC,mBAAmBC,KAAKC,UAAUb,GACrDQ,GAAGM,IAAMlD,KAAKsB,SAAWwB,EAC3B,MAAOK,GACD9D,GAAUA,EAAO+D,SAAyC,kBAAvB/D,GAAO+D,QAAQC,MAClDD,QAAQC,IAAI,uDAAyDF,GACrEC,QAAQC,IAAI,mBAAoBjB,MAO5CD,WAAY,WACR,GAAImB,GAAShE,EAASgE,OAClBC,EAAID,EAAOE,QAAQ3B,EACvB,IAAO,EAAJ0B,EACC,OAAO,CAEP,IAAIE,GAAMH,EAAOE,QAAQ,IAAKD,EAAI,EAElC,OADAE,GAAY,EAANA,EAAUH,EAAOvB,OAAS0B,EACzBH,EAAOI,MAAMH,EAAIzB,EAA2B2B,IAG3DvB,UAAW,SAASyB,GAChBrE,EAASgE,OAASzB,EAAqB,IAAM8B,GAIrD,IAAIC,GAAWvE,EAAOwE,UAElBzD,EAAU,GAAIL,EAElB,IAAG6D,GAAYA,EAAS7B,OAAS,CAC7B,GAAIwB,GAAI,EACJO,EAAUF,EAAS7B,MACvB,KAAIwB,EAAI,EAAOO,EAAJP,EAAaA,IACpBnD,EAAQU,KAAK8C,EAASL,IAI9BlE,EAAOwE,UAAYzD,EAEnBf,EAAOU,cAAgBA,GAExBV,OAAQC"}
```

Comment:
@psquickitjayant was the latest map file checked in last time you did a commit? I wouldn't block the merge though since we don't serve source maps currently.